src/core/Timepicker/BaseTimePicker.tsx

```diff
@@ -0,0 +1,26 @@
+import 'date-fns';
+import React, { useState } from 'react';
+import MomentUtils from '@date-io/moment';
+import { MuiPickersUtilsProvider, TimePicker } from '@material-ui/pickers';
+import { BaseTimePickerProps } from './base';
+
+interface TimePickerProps extends BaseTimePickerProps {
+  value?: Date;
+}
+
+const BaseTimePicker: React.FC<TimePickerProps> = ({ value }) => {
```

Comment:
```suggestion
const TimePicker: React.FC = () => {
```

Reply:
@S-ayanide no need to pass this value via props?

Reply:
Date is not required to be passed as props